docker/images/test-runner/Dockerfile

```diff
@@ -125,10 +125,9 @@ RUN : \
   ; /compile_protobuf.sh \
   ; :
 
-COPY sigoptcompute/sigoptcompute /sigopt-server/sigoptcompute/sigoptcompute
-COPY sigoptaux/sigoptaux /sigopt-server/sigoptaux/sigoptaux
+COPY libsigopt /sigopt-server/libsigopt
 
-ENV PYTHONPATH /sigopt-server/src/python:/sigopt-server/sigoptcompute:/sigopt-server/sigoptaux:/sigopt-server
+ENV PYTHONPATH /sigopt-server/src/python:/sigopt-server/compute:/sigopt-server
```

Comment:
so is `compute` here an alias or something? I would thought this had to be something like `sigopt-server/libsigopt`?

Reply:
no I just need to remove it, good catch